src/Data/Bifunctor/Reverse.hs

```diff
@@ -0,0 +1,43 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE PolyKinds #-}
+{-# LANGUAGE Trustworthy #-}
+-- |
+-- Copyright   :  (C) 2021 David Feuer
+-- License     :  BSD-style (see the file LICENSE)
+-- Maintainer  :  Edward Kmett <ekmett@gmail.com>
+-- Stability   :  provisional
+-- Portability :
+--
+-- Making bifunctors whose elements are notionally in the
+-- reverse order from the original bifunctor.
+
+module Data.Bifunctor.Reverse
+  ( Reverse (..)
+  ) where
+
+import Data.Biapplicative
+import qualified Data.Bifunctor as Base
+import GHC.Generics (Generic, Generic1)
+import qualified Data.Functor.Reverse as FunReverse
+import Control.Applicative.Backwards
+import Data.Coerce
+import Data.Bifoldable (Bifoldable (..))
+import Data.Bitraversable (Bitraversable (..))
+import Data.Semigroup (Dual (..))
+
+newtype Reverse t a b = Reverse { getReverse :: t a b }
```

Comment:
It would be helpful to include some Haddocks which describe the role that `Reverse` plays with respect to its `{Bi}foldable` and `{Bi}traversable` instances, similar to the Haddocks you've given for `Backwards`.